admin-ui/app/styles/custom/_typography.scss

```
@@ -1,5 +1,5 @@
 // Typography variables corresponding to app/styles/fonts.ts
-$font-family-primary: "Mona-Sans", sans-serif;
+$font-family-primary: Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Helvetica" to be "helvetica" (value-keyword-case)

(value-keyword-case)